src/code/z_view.c

```diff
@@ -51,80 +51,88 @@ void View_Init(View* view, GraphicsContext* gfxCtx) {
     view->up.z = 0.0f;
     view->eye.z = -1.0f;
 
-    if (D_8012ABF0) {
-        if (D_8012ABF0 == 0) {}
+    if (sViewNotInitialized) {
+        if (sViewNotInitialized == 0) {}
         osSyncPrintf("\nview: initialize ---\n");
-        D_8012ABF0 = false;
+        sViewNotInitialized = false;
     }
 
     view->unk_124 = 0;
-    view->flags = 1 | 2 | 4;
+    view->dirtyFlags = VIEW_VIEWING | VIEW_VIEWPORT | VIEW_PROJECTION_PERSPECTIVE;
     View_InitDistortion(view);
 }
 
-void func_800AA358(View* view, Vec3f* eye, Vec3f* lookAt, Vec3f* up) {
+void View_LookAt(View* view, Vec3f* eye, Vec3f* lookAt, Vec3f* up) {
     if (eye->x == lookAt->x && eye->z == lookAt->z) {
         eye->x += 0.1f;
     }
 
     view->eye = *eye;
     view->lookAt = *lookAt;
     view->up = *up;
-    view->flags |= 1;
+    view->dirtyFlags |= VIEW_VIEWING;
 }
 
-void func_800AA3F0(View* view, Vec3f* eye, Vec3f* lookAt, Vec3f* up) {
+/*
+ * Unused. View_LookAt is always used instead. This version is similar but
+ * is missing the input sanitization and the update to the dirtyFlags.
+ */
+void View_LookAtUnsafe(View* view, Vec3f* eye, Vec3f* lookAt, Vec3f* up) {
```

Comment:
What is this concept of sanitazation
```suggestion
/*
 * Unused. View_LookAt is always used instead. This version is similar but
 * is missing the input sanitization and the update to the dirtyFlags.
 */
void View_SetEyeAtUp(View* view, Vec3f* eye, Vec3f* lookAt, Vec3f* up) {
```
Could this not be called this? I'm not sure about this whole concept of input sanitization and security. It seems like a bit of a stretch to interpret it this way

Reply:
Yeah, there's no security stuff here.

It's sanitization in that it quietly corrects its inputs if they would lead to a divide-by-zero (eye == at).

Reply:
See also long discussion here on a name for this https://github.com/zeldaret/oot/pull/1191#discussion_r837722332